cypress/integration/RichTextInput/Rich_Text_Toolbar.feature

```diff
@@ -0,0 +1,68 @@
+Feature: Rich text toolbar
+
+  Scenario Outline: The user clicks formatting buttons with text selected
+    Given there is some text selected
```

Comment:
I think this should be `Given there is some plain text selected` (either `plain` or `unformatted`).

Reply:
Indeed, that's important for these to work. Added to all scenario where appropriate.